src/layer/BYOCLayer.ts

```diff
@@ -33,6 +34,7 @@ interface ConstructorParameters {
   description?: string | null;
   collectionId?: string | null;
   locationId?: LocationIdSHv3 | null;
+  subType?: BYOCSubTypes | null;
```

Comment:
I'm not sure how SH really calls this. I named it `subType` because this how `https://services.sentinel-hub.com/configuration/v1/wms/instances/{instanceId}/layers` calls it. 

In `/process` for example, `type` refers to the `{subType}-{collectionId}`.

So `collectionType` is a valid option imo, but I'd prefer to use something term which is used somewhere, that we get our info from.

What did you mean with "ffti"

Reply:
ok, if `subType` is used then it is probably good enought. Feel free to ignore :)